packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -5475,7 +4734,7 @@ describe('NetworkController', () => {
         await expect(async () =>
           controller.upsertNetworkConfiguration(
             {
-              chainId: '0x9999',
+              chainId: '0x1',
```

Comment:
Nit: This change is not necessary:
```suggestion
              chainId: '0x9999',
```